onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -751,6 +751,35 @@ def sample_inputs_unfold(op_info, device, dtype, requires_grad, **kwargs):
     yield opinfo_core.SampleInput(t, args=(dimension, size, step))
 
 
+def sample_inputs_slice_scatter(op_info, device, dtype, requires_grad, **kwargs):
+    del op_info
+    del kwargs
+    make_arg = functools.partial(
+        torch_testing.make_tensor, dtype=dtype, device=device, requires_grad=requires_grad
+    )
+
+    L = 20
+    cases = (
+        ((L, L, L), (L, L, L), (0, 0, L, 1)),
+        ((L, L, L), (L // 2, L, L), (0, L // 2, L, 1)),
+        ((L, L, L), (L // 4, L, L), (0, L // 2, L, 2)),
+        ((L, L, L), (L, L, L), (1, 0, L, 1)),
+        ((L, L, L), (L, L // 2, L), (1, L // 2, L, 1)),
+        ((L, L, L), (L, L // 4, L), (1, L // 2, L, 2)),
+        ((L, L, L), (L, L, L), (2, 0, L, 1)),
+        ((L, L, L), (L, L, L // 2), (2, L // 2, L, 1)),
+        ((L, L, L), (L, L, L // 4), (2, L // 2, L, 2)),
+        ((L, L, L), (L, L // 2, L), (1, L // 2, L * 2, 1)),  # end > L
+        ((L, L, L), (L, L, L), (-2, 0, L, 1)),  # negative dim
+        ((L, L, L), (L, L, L // 4), (-1, L // 2, L * 2, 2)),  # end > L and negative dim
```

Comment:
Nice comments, Thanks! Would you be willing to add these cases to pytorch as well?

Reply:
> Nice comments, Thanks! Would you be willing to add these cases to pytorch as well?

I would rather not do this as I am not quite familiar with the process. You guys can do this. :-)

Reply:
The process is simple. It is just a matter of adding these three lines in a PR.